src/views/renewal-email-success.njk

```diff
@@ -24,29 +24,20 @@
 
   <h2 class="govuk-heading-m">Contact us</h2>
 
+  <p class="govuk-body">Contact our Licensing team if you need further assistance.</p>
+
   <h3 class="govuk-heading-s">Email</h3>
 
-  <ul class="govuk-list">
-    <li>
-      <a class="govuk-link" href="mailto:licensing@nature.scot">licensing@nature.scot</a>
-    </li>
-    <li>
-      We aim to respond within 2 working days.
-    </li>
-  </ul>
+  <p class="govuk-body"><a class="govuk-link" href="mailto:licensing@nature.scot">licensing@nature.scot</a></p>
 
   <h3 class="govuk-heading-s">Telephone</h3>
 
-  <ul class="govuk-list">
-    <li>
-      01463 725 364
-    </li>
-    <li>
-      Voicemail service. We aim to respond within 2 working days.
-    </li>
-  </ul>
+  <p class="govuk-body">01463 725 364</P>
+
+  <p class="govuk-body">Monday to Friday, 10am to 4pm (except public holidays)
```

Comment:
line 35 has a capitalised closing tag, line 37 is missing the closing paragraph tag

Reply:
Those tags have been updated :) great spot!!!